scenarios.md

```diff
@@ -262,6 +261,20 @@ A user may want to trust an artifact only if it is signed by a specific key. May
 
 1. Users must be allowed to configure specific trusted keys for specific artifacts or collections of artifacts.
 
+### Scenario #11: Revoking a Signature
+
+A signer determines that a signed artifact is no longer trusted. This could be a result of accidentally signing the wrong artifact, or a CVE discovered in the image and patched in a newer release. It should be possible for the signer to revoke the trust in that vulnerable artifact.
+
+1. A CVE is discovered in a signed artifact.
+1. The signature on the vulnerable artifact is revoked.
+1. Consumers that pulled the vulnerable artifact that verify the signature will receive an error indicating the signer no longer trusts the artifact.
```

Comment:
Adding in feedback based in discussions we had in [Rescinding signatures](https://hackmd.io/2rc07WwHTdmPjpgoMbvYfA?view)

The user may receive an error or warning based on their configuration for handling revocations. How about - _Consumers that pulled the vulnerable artifact that verify the signature will receive an error or warning indicating the signer no longer trusts the artifact, based on the policy the Consumer defines._

Reply:
Good point, but I guess it depends on where the boundaries are in the scenario. Notary itself should report an error and only give a pass/fail. Tooling that checks Notary for the admission controller, container runtime, etc, may interpret that error as a warning or block the action.

Reply:
Would it suffice to say that whatever API lists the current valid tags for an image not list the revoked ones?  But not try to 'reject' using an image which has been revoked?  (Sorry, I'm not sure where the current detailed API list lives)

Reply:
@hallyn since the tag could have been previously pulled, I don't think a registry side API to remove it from the list would suffice. That would also require the tag list to be signed to avoid a malicious actor pushing a bad tag list.

@gokarnm I've updated the PR to define this as a "notification" which the client could decide different ways to handle.